README.md

```diff
@@ -26,6 +27,33 @@ Install from source
 ```bash
 git clone git@github.com:MobleyLab/blues.git
 python setup.py install
+=======
+## Installation
+### Prerequisites - Anaconda
+*We recommend installing the full [Anaconda](https://www.continuum.io/downloads) distribution.* If you are low on disk space, you can use [miniconda](http://conda.pydata.org/miniconda.html).
+
+### Package dependencies
+Install the required packages into a new environment
+
+```
+conda create -c omnia -c omnia/label/dev -n blues python=3.5 openmm==7.0.1 openmmtools alchemy mdtraj
```

Comment:
(Oh, I see, this is creating a blues environment. Sorry, my bad. But the question on OpenMM still stands.)